.github/workflows/testGithubActionsWorkflows.yml

```diff
@@ -0,0 +1,35 @@
+name: Test GitHub Actions workflows
+
+on:
+  workflow_dispatch:
+  workflow_call:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+    branches-ignore: [staging, production]
+    paths: ['.github']
```

Comment:
This workflow should have run in this PR also. But I don't see it here.

I think it might be our path?
```suggestion
    paths: ['.github/**']
```


<img width="868" alt="image" src="https://github.com/Expensify/App/assets/29673073/6f5a9cd2-e15e-42e2-bf31-36c416546743">

Reply:
Yes, you're probably right

Reply:
Updated